examples/node/package.json

```diff
@@ -1,5 +1,4 @@
 {
-  "name": "node",
```

Comment:
Is there a real reason to remove it from the example?

Reply:
No, was just testing. I can revert